phys2bids/viz.py

```diff
@@ -20,7 +21,7 @@ def time2ntr(x):
         return x / options.tr
 
     def ntr2time(x):
-        return x * options.tr
+        return x * options.trprint(f'saving channels plot at plot at {options.chplot}')
```

Comment:
I think there's a mistake here?
the print should go two lines later one indentation level less!

Reply:
actually i don't know how that happened but is taken care of